cmd/metal-api/internal/service/switch-service.go

```diff
@@ -1026,7 +1044,7 @@ func findSwitchReferencedEntities(s *metal.Switch, ds *datastore.RethinkStore) (
 	return p, ips.ByProjectID(), m, ss, nil
 }
 
-func makeSwitchResponseList(ss metal.Switches, ds *datastore.RethinkStore) ([]*v1.SwitchResponse, error) {
+func (r *switchResource) makeSwitchResponseList(ss metal.Switches, ds *datastore.RethinkStore) ([]*v1.SwitchResponse, error) {
```

Comment:
for `makeSwitchResponse` you dropped the datastore parameter. Is there a reason not to do the same for `makeSwitchResponseList`?

Reply:
Ah thanks, one moment

Reply:
fixed